tests/test_pkgset.py

```diff
@@ -1,22 +1,29 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
-
+from __future__ import print_function
 
 import unittest
 import run_tests # set sys.path
 
+import warnings
 import os
+import sys
 import tempfile
 import shutil
 import hashlib
 import cPickle as pickle
-import rpm
-import warnings
-
-from kobo.pkgset import *
-
 
-class TestFileWrapperClass(unittest.TestCase):
+try:
+    import rpm
+except ImportError:
+    print('SKIP: rpm module is not available.', file=sys.stderr)
+    # Changing test superclass effectively skips the tests.
+    TEST_CLASS = object
+else:
+    from kobo.pkgset import *
+    TEST_CLASS = unittest.TestCase
```

Comment:
I think that I've been reluctant to pull in any new deps, but for no good reason that I can think of.  So I'll do that, it should be better.